presto-docs/src/main/sphinx/connector/teradata.rst

```diff
@@ -0,0 +1,53 @@
+===================
+Teradata  Connector
+===================
+
+Teradata provides a low latency high performing connector that
+supports high concurrency, and parallel processing between Teradata
+and Presto. The Teradata QueryGrid connector for Presto enables users
```

Comment:
We should mention QueryGrid at some point, but maybe just call it the Teradata connector for now (that's more familiar to Presto users)

suggest; "The Teradata connector for Presto enables users to query Teradata from Presto. The connector is architected..."
e.g. for this top bit, take out the T2P blurb.